app/src/main/java/chat/rocket/android/chatroom/adapter/BaseViewHolder.kt

```diff
@@ -65,7 +65,9 @@ abstract class BaseViewHolder<T : BaseUiModel<*>>(
                 val manager = FlexboxLayoutManager(context, FlexDirection.ROW)
                 recyclerView.layoutManager = manager
                 recyclerView.adapter = adapter
-                adapter.addReactions(it.reactions.filterNot { it.unicode.startsWith(":") })
+                adapter.addReactions(it.reactions.filterNot {
+                    it.unicode.startsWith(":") && it.url.isNullOrEmpty()
```

Comment:
Better to call `reactionUiModel`here (due to the nested lambda w/ implicit parameter)?

```
adapter.addReactions(it.reactions.filterNot { reactionUiModel ->
  reactionUiModel.unicode.startsWith(":") && reactionUiModel.url.isNullOrEmpty() 
}
```